### ISSUE TITLE
[Bug]: Import pymilvus gets error "module 'pymilvus' has no attribute 'bulk_load'"

### ISSUE THREAD

**Issue by jaelgu** [milvus-io/pymilvus#1076]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

`from pymilvus import *` get error "module 'pymilvus' has no attribute 'bulk_load'" with v2.1.0:

<img width="506" alt="image" src="https://user-images.githubusercontent.com/86251631/181466441-d567f843-85e7-4b3c-9b33-8fb1e44551d5.png">

- it was good in v2.0.2
- reproducible in both Linux & Mac

### Expected Behavior

No error out when `from pymilvus import *`

### Steps/Code To Reproduce behavior

```markdown
1. Pip install pymilvus 2.1.0
2. Run in python: 

from pymilvus import *
```
```


### Environment details

```markdown
- Hardware/Softward conditions (OS, CPU, GPU, Memory): CPU
- Method of installation (Docker, or from source): Docker
- Milvus version (v0.3.1, or v0.4.0): 2.1.0
- Milvus configuration (Settings you made in `server_config.yaml`):
```

**Comment by XuanYang-cn** [comment]
fixed at pymilvus-2.1.1.dev2
/reopen

**Comment by binbinlv** [comment]
Fixed in 2.1.1.dev2:
```
>>> from pymilvus import *
>>>
```
